Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Driver.hs

```diff
@@ -845,9 +850,10 @@ respondQuote driverId req = do
     sendRemoveRideRequestNotification driverSearchReqs orgId driverQuote = do
       for_ driverSearchReqs $ \driverReq -> do
         Esq.runNoTransaction $ do
+          --TODO: Why?
           QSRD.updateDriverResponse driverReq.id Pulled
```

Comment:
> for analytics.

I meant `runNoTransaction`

Reply:
Yeah it can but it is there avoid db deadlocks, as this SearchRequestForDriver table is being used many places, so we were having deadlock on this tables. Hence remove transaction block here.